tests/plugins/cloudservices/cloudservices.js

```diff
@@ -101,18 +114,6 @@
 			listener.removeListener();
 		},
 
-		'test no TOKEN error': function() {
```

Comment:
I don't understand why this test was removed as `no-token` error is still present in editor.

Reply:
It was no longer viable in the former implementation and would have to be rewritten.